### ISSUE TITLE
UI Change: slider bars

### ISSUE THREAD

**Issue by scienceworld** [ScienceWorldCA/domelights#17]
The slider bars need work. I’d remove the border and make the slider bar a light grey with a simple circle for the handlebar

**Comment by jporterscienceworld** [comment]
I think the left/right arrows should move the dome a bit not just start rotating it in that direction. Maybe just 15% at a time. Not sure if you even need the slider. You can compact the controls so the pause/play are between the arrows and centered above the timeline control.

**Comment by RobertButterworth** [comment]
Yep can do. I do like the slider, you can give a big spin and run your finger over the dome to paint but there might be a better way to do that.

**Comment by RobertButterworth** [comment]
With the latest update (V.0.2.3) the slider still has the functionality and design of both methods. Do you still want me to change the left and right to only rotate by 15%?
